larray/viewer/model.py

```diff
@@ -277,6 +288,19 @@ def set_format(self, format):
         self._format = format
         self.reset()
 
+    def _index_to_position(self, index):
+        """
+        Cell at position (0, 0) contains the first data.
```

Comment:
the first data cell? (data is not either an adjective or uncountable noun in English)